app/views/layouts/no_ember.html.erb

```diff
@@ -22,6 +22,7 @@
     </div>
   </section>
   <%= theme_lookup("footer") %>
+  <%= raw theme_lookup("body_tag") %>
```

Comment:
Do you need the `raw` here? If you need, why isn't it needed for the `footer` and in the `crawler` template?

```suggestion
  <%= theme_lookup("body_tag") %>
```